neo/rawio/axonarawio.py

```diff
@@ -558,6 +557,20 @@ def get_active_tetrode(self):
                     tetrode_id = int(key.strip("collectMask_"))
                     active_tetrodes.append(tetrode_id)
         return active_tetrodes
+    
+    def get_active_channels(self):
+        """
+        Returns the ID numbers of the active channels as a list.
+        E.g.: [20,21,22,23] for tetrode 6 active.
+        """
+        active_tetrodes = self.get_active_tetrode()
+        active_channels = []
+        
+        for tetrode in active_tetrodes:
+            chan = self._get_channel_from_tetrode(tetrode)
+            active_channels.append(chan)
+        
+        return np.concatenate(active_channels)
```

Comment:
what's the benefit of concat? I need to doublecheck what is `chan` is it a list of all channels from a tetrode. If that is the case I would call it `channels` or `chans` instead so we know it's plural.